dev/KaairaGupta/Ml_functions.py

```diff
@@ -0,0 +1,28 @@
+
+from data_loader import load_split_preprocessed_data
+from sklearn.metrics import confusion_matrix,recall_score,precision_recall_curve,auc,roc_curve,roc_auc_score,classification_report
+import warnings
+import matplotlib.pyplot as plt # to plot graph
+import seaborn as sns
+warnings.filterwarnings('ignore')
+
+#model function
+def model(model,features_train,features_test,labels_train,labels_test):
```

Comment:
nit: Please include a full doc-string explaining the contents of `model`

Reply:
I have included as required. Also, commented most of the code.
Thanks!